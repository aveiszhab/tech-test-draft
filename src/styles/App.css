.App {
    display: flex;
    flex-direction: column;
}

.nasa-logo {
    align-self: center;
    width: 28vw;
}