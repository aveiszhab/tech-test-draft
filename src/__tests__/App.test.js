import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../components/App';

describe('App', () => {
    it('renders correctly to match the App snapshot', () => {
        const {asFragment} = render(<App />);
  
        expect(asFragment).toMatchSnapshot();
    });

    it('renders App correctly', () => {
               
      render(<App/>);
      const image = screen.getByRole('img', {name: /nasa/i});
  
      expect(image).toHaveClass('nasa-logo');
      expect(screen.getByAltText('NASAlogo')).toBeInTheDocument();
      
      });
});