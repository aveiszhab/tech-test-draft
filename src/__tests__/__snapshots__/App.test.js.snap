// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly to match the App snapshot 1`] = `[Function]`;
